src/main/java/de/tum/in/www1/artemis/service/connectors/bamboo/BambooBuildPlanService.java

```diff
@@ -287,13 +288,16 @@ private BitbucketServerRepository createBuildPlanRepository(String name, String
                 .changeDetection(new VcsChangeDetection());
     }
 
-    private PlanPermissions generatePlanPermissions(String bambooProjectKey, String bambooPlanKey, @Nullable String teachingAssistantGroupName, String instructorGroupName,
-            String adminGroupName) {
+    private PlanPermissions generatePlanPermissions(String bambooProjectKey, String bambooPlanKey, @Nullable String teachingAssistantGroupName, @Nullable String editorGroupName,
+                                                    String instructorGroupName, String adminGroupName) {
         var permissions = new Permissions().userPermissions(bambooUser, PermissionType.EDIT, PermissionType.BUILD, PermissionType.CLONE, PermissionType.VIEW, PermissionType.ADMIN)
                 .groupPermissions(adminGroupName, PermissionType.CLONE, PermissionType.BUILD, PermissionType.EDIT, PermissionType.VIEW, PermissionType.ADMIN)
                 .groupPermissions(instructorGroupName, PermissionType.CLONE, PermissionType.BUILD, PermissionType.EDIT, PermissionType.VIEW, PermissionType.ADMIN);
+        if (editorGroupName != null) {
+            permissions = permissions.groupPermissions(editorGroupName, PermissionType.CLONE, PermissionType.BUILD, PermissionType.EDIT, PermissionType.VIEW, PermissionType.ADMIN);
+        }
         if (teachingAssistantGroupName != null) {
-            permissions = permissions.groupPermissions(teachingAssistantGroupName, PermissionType.BUILD, PermissionType.EDIT, PermissionType.VIEW);
+            permissions = permissions.groupPermissions(teachingAssistantGroupName, PermissionType.VIEW);
```

Comment:
Teaching assisstants are not allowed to build and view the build plan?

Reply:
They are allowed to view it, but not to build it - is there a use case where a tutor should be able to build it?

Reply:
As far as I remember we wanted to prevent that the manual assessment can be overridden - I personally never used it as a tutor, but if that's a common use case e.g. in PGdP, we should allow it.

Reply:
I honestly am not sue either

Reply:
I would then leave it as it is with the reasoning that tutors could otherwise override manual assessments and since they can't modify build plans there should be no reason why they need to execute it